frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -60,6 +71,24 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
                 },
             ],
         ],
+        uniqueCreators: [
+            (selectors) => [selectors.featureFlags],
+            (featureFlags) => {
+                const creators = {}
+                featureFlags.forEach((flag: FeatureFlagType) => {
+                    if (flag.created_by) {
+                        if (!creators[flag.created_by.id]) {
+                            creators[flag.created_by.id] = flag.created_by.first_name
+                        }
+                    }
+                })
+                const response: LemonSelectOption<string>[] = [{ label: 'Any user', value: 'any' }]
+                for (const [id, first_name] of Object.entries(creators)) {
+                    response.push({ label: first_name, value: id } as LemonSelectOption<string>)
+                }
+                return response
```

Comment:
Specifying ` as LemonSelectOption<string>` can hide subtle bugs. TypeScript should be smart enough to infer what type the data you're pushing to `response` should be.

Normally you shouldn't need to type any variable more than once.

Also, if you want to get fancy, here's another way to do this:

```ts
const response: LemonSelectOption<string>[] = [
    { label: 'Any user', value: 'any' },
    ...(Object.entries(creators).map(
        ([id, first_name]) => ({ label: first_name, value: id }),
    )),
]
```